Gemfile

```diff
@@ -37,6 +37,9 @@ gem 'paper_trail-globalid'
 gem 'devise', '~> 4.7'
 gem 'omniauth-google-oauth2', '~> 0.8.0'
 
+# Strong Password for user password validation for folks not on oauth
+gem 'strong_password', '~> 0.0.9'
```

Comment:
I tried the look at the source for this version, and found that the dev actually hasn't published the tag yet: https://github.com/bdmac/strong_password/issues/38

This is not a problem with your PR (0.0.9 is the latest version published on rubygems), just something to be aware of.